tests/tutor/test_tutor_teacher.py

```diff
@@ -0,0 +1,1473 @@
+    # GIVEN: Logged into Tutor as a teacher
+    # AND: Has an existing course
+    # AND: Has at least two students
+    # AND: Has a graded assignment
+
+    # WHEN: If the user has more than one course, click on a Tutor course name
+    # AND: Click on the ""Student Scores"" button
+    # AND: Click ""Score""
+
+    # THEN: Students are sorted by completion of selected assignment.
+
+
+@expected_failure
+@test_case('')
+@tutor
+def test_edit_score_weights(tutor_base_url, selenium, teacher):
+    """Edit score weights as teacher."""
+    # GIVEN: Logged into Tutor as a teacher
+    # AND: Has an existing course
+
+    # WHEN: If the user has more than one course, click on a Tutor course name
+    # AND: Click on the "Student Scores" button
+    # AND: Click ""set weights"" on the page
+
+    # THEN: A window for edit weights should pop up
+    # AND: user should be able to change the weights from this window
+
+
+@expected_failure
+@test_case('')
+@tutor
+@nondestructive
+def test_cancel_edit_score_weights(tutor_base_url, selenium, teacher):
+    """Cancel editing score weights as teacher."""
+    # GIVEN: Logged into Tutor as a teacher
+    # AND: Has an existing course
+
+    # WHEN: If the user has more than one course, click on a Tutor course name
+    # AND: Click on the "Student Scores" button
+    # AND: Click "set weights" on the page
+    # AND. click "set weights" on the page
+    # AND.  change some weights in the popup window
+    # AND. click cancel button
+
+    # THEN: User is taken back to the scores page, nothing should change
+
+
+@expected_failure
+@test_case('')
+@tutor
+@nondestructive
+def test_click_see_why_in_score_weights(tutor_base_url, selenium, teacher):
+    """Go to "see why" blog page from score weights as teacher"""
+    # GIVEN: Logged into Tutor as a teacher
+    # AND: Has an existing course
+
+    # WHEN: If the user has more than one course, click on a Tutor course name
+    # AND: Click on the "Student Scores" button
+    # AND: click "set weights" at student scores page
+    # AND: click "see why" from the set weights page
+
+    # THEN: the openstax blog should open up in another tab"
+
+
+@expected_failure
+@test_case('')
+@tutor
+@nondestructive
+def test_scrolling_in_assignments_section(tutor_base_url, selenium, teacher):
+    """ScrolL left and right in assignment section in score page."""
+    # GIVEN: Logged into Tutor as a teacher
+    # AND: Has an existing course
+
+    # WHEN: Click on the ""Student Scores"" button
+    # AND: Scroll left or right in the table of scores for view assignments
+
+    # THEN: Table should move accordingly with the scrolling"
+
+
+@expected_failure
+@test_case('')
+@tutor
+@nondestructive
+def test_score_page_for_section_with_no_scores(tutor_base_url, selenium,
+                                               teacher):
+    """Test proper messages in score page for a section that has no score to
+    display."""
```

Comment:
If you need more than a single line for the doctext, shorten the imperative statement to just one line, add a blank line, then include a more detailed summary for the function. In this case, because it's a test function, go ahead and shorten the text.